recipes/ldraw/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set parts_version = "1902" %}
+{% set mklist_version = "1.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: ldraw
+
+source:
+  - url: http://www.ldraw.org/library/updates/complete.zip
+    sha256: 57a539df42fa70875af7a6eef6459b2101aa7269d9c157ab3421f019797c7b99
+
+  - url: http://www.ldraw.org/library/official/LDConfig.ldr
+    sha256: 4d5e1a2416aea0db5af5ad67fc768dd461401607f46ebe57331f397a493f5ceb
+
+build:
+  number: {{ build_number }}
+
+outputs:
+  - name: ldraw-parts
+    version: {{ parts_version }}
+    script: build_parts.sh   # [unix]
+    script: build_parts.bat  # [win]
+    test:
+      commands:
+        - test -d $PREFIX/share/ldraw/parts/           # [unix]
+        - test -d $PREFIX/share/ldraw/models/          # [unix]
+        - test -f $PREFIX/share/ldraw/models/car.ldr   # [unix]
+        - test -f $PREFIX/share/ldraw/parts/u9462.dat  # [unix]
+        - if exist %LIBRARY_PREFIX%\share\ldraw\parts\u9462.dat (exit 0) else (exit 1)  # [win]
+        - if exist %LIBRARY_PREFIX%\share\ldraw\models\car.ldr (exit 0) else (exit 1)   # [win]
+
+  - name: ldraw-mklist
+    version: {{ mklist_version }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - unzip      # [not win]
+        - m2-unzip   # [win]
+        - m2-sed     # [win]
```

Make should be installed from conda instead of relying on the host:
```suggestion
        - m2-sed     # [win]
        - make
```
(for example the POWER PC builds don't have `make` installed)